recipes/or-tools/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -BUILD_DEPS=OFF \
+      -DUSE_SCIP=OFF \
+      $SRC_DIR
+
+make -j${CPU_COUNT}
+make install
+
+# run tests
+make test
```

Comment:
again here:
```
cmake --build <build_dir> --target test
```
or if you want to filter then you can call ctest from the build dir directly
e.g. to run only "cxx_*" tests
```sh
(cd build_dir && ctest -R "^cxx_")
```

Reply:
I will have a look at 1. and 2. but RE 3.: Conda supports a [test section](https://docs.conda.io/projects/conda-build/en/latest/concepts/recipe.html#testing) in its meta.yaml (Which is sort of the installation instruction).
This section creates a new environment during the build and installs the package and its dependencies in it. Once installed, normal commands can then be run. For Python packages that are normally `pytest test` but it would probably make sense to find something similar here. 
One note: conda-forge doesn't need the full test suite to be run. As long as we can verify that the package works it's fine.

Reply:
Quick note: This can only be tackled after the overlinking error.